### PR TITLE
Doc: Improved Troubleshooting section on README 

### PR DESCRIPTION
**Pull request by adorilson**
This PR improved Troubleshooting section on README by:

- Added instruction to add user to docker group to resolv non-specific errors
- Added instruction to restore pip if need
- Removed a ambiguity in a subsection title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +79,29 @@ SPIDER=sc_florianopolis make run_spider
 
 ## Troubleshooting
 
-### "Permission denied" error when files are downloaded
+### Non-specific errors when building or running
+Several error is because your user don't is from docker group. To resolve this,
```
Comment (he7d3r) [inline README.md:83]: "don't is from" -> "is not in the"?

File: README.md [hunk 2]
```diff
@@ -74,7 +79,29 @@ SPIDER=sc_florianopolis make run_spider
...
+execute the follow command and do logout and login:
+
+```console
+$ sudo usermod -aG docker $USER
+```
```
Comment (jvanz) [inline README.md:88]: Add a sentence to remember the users that they need to logout and login in the session

**Review by jvanz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (adorilson) [reply README.md:83]: I'll fix. Txs.

File: README.md (hunk 2 above)
Comment (adorilson) [reply README.md:88]: The previous line is:

`execute the follow command and do logout and login`

It is not enough?

File: README.md (hunk 1 above)
Comment (jvanz) [inline README.md:83]:
```suggestion
Several error is because your user is not in the docker group. To resolve this,
```

**Review by jvanz — APPROVED** [review]

**Pull request merged**